pkgs/shells/fish/default.nix

```diff
@@ -306,40 +291,6 @@ let
     };
 
     passthru = {
-      shellPath = "/bin/fish";
```

Comment:
Shells must have `shellPath` or they will not be recognized by NixOS!

Reply:
Fixed.